---
title: "CovidExpress"
layout: textlay
excerpt: "- CovidExpress"
sitemap: false
permalink: /covidexpress/
---
## [CovidExpress Server](https://covidexpress.herokuapp.com){:target="_blank"}
---

[![]({{ site.url }}{{ site.baseurl }}/images/codepic/cellxgene_gsea.png){: style="width: 250px; float: right;margin-left: 30px; margin-right: 30px; border: 10px"}]({{ site.url }}{{ site.baseurl }}/images/codepic/cellxgene_gsea.png)
#### CovidExpress: an integrative database and portal for processed RNA-seq data allow easy investigation for SARS-CoV-2 related research

#### Abstract
<div style="text-align: justify">
Infection with Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) in human could cause Coronavirus disease 2019 (COVID-19). Since its first discovery in Dec 2019, SARS-CoV-2 has become a global pandemic and caused 2.6 million direct/indirect deaths. Scientific community responded to COVID-19 quickly and data sharing played essential role across all aspects of combatting against SARS-CoV-2. However, to date, only few databases allow access to expression levels in bulk RNA-seq for limited samples, and none of them allow easy access of analysis for researchers without programming skills. To fill this gap, we collected about 1500 samples from publicly available resources and developed a database and visualization tool CovidExpress (https://stjudecab.github.io/covidexpress)
</div>
---- 
#### [CovidExpress Expression Data Download(.h5)](https://covidexpress.herokuapp.com){:target="_blank"}
---- 
#### Clustering Results for reviewing and Download (see below for details)
<table style="border-spacing: 10px;" border="1" class="dataframe table-bordered">
<thead>
<tr style="text-align: center;">
<th>Score</th>
<th>ssGSEA</th>
<th>ssGSEA</th>
<th>ssGSEA</th>
<th>FPKM</th>
<th>FPKM</th>
</tr>
<tr style="text-align: right;">
<th>Data</th>
<th>COV.Significant</th>
<th>COV.All.Signature</th>
<th>MSigDB</th>
<th>BatchCorrected</th>
<th>Original</th>
</tr>
</thead>

<tbody>
<tr>
<th>PCA.3D</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.PCA.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.PCA.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.PCA.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.PCA.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.PCA.3D.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>PCA.2PCs</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.PCA.2PCs.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.PCA.2PCs.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.PCA.2PCs.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.PCA.2PCs.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.PCA.2PCs.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>PCA.grid</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.PCA.grid.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.PCA.grid.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.PCA.grid.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.PCA.grid.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.PCA.grid.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>PCA.components_matrix</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.PCA.components_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.PCA.components_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.PCA.components_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.PCA.components_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.PCA.components_matrix.tsv" target="_blank">Download tsv</a></td>
</tr>
<tr>
<th>PCA.loadings</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.PCA.loadings.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.PCA.loadings.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.PCA.loadings.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.PCA.loadings.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.PCA.loadings.tsv" target="_blank">Download tsv</a></td>
</tr>
<tr>
<th>PCA.loadings_matrix</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.PCA.loadings_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.PCA.loadings_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.PCA.loadings_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.PCA.loadings_matrix.tsv" target="_blank">Download tsv</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.PCA.loadings_matrix.tsv" target="_blank">Download tsv</a></td>
</tr>
<tr>
<th>tSNE.3D</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.tSNE.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.tSNE.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.tSNE.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.tSNE.3D.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.tSNE.3D.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>tSNE.2D</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.tSNE.2nc.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.tSNE.2nc.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.tSNE.2nc.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.tSNE.2nc.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.tSNE.2nc.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>UMAP.3D.n_neighbors_20</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.UMAP.3D.n_neighbors_20.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.UMAP.3D.n_neighbors_20.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.UMAP.3D.n_neighbors_20.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.UMAP.3D.n_neighbors_20.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.UMAP.3D.n_neighbors_20.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>UMAP.3D.n_neighbors_10</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.UMAP.3D.n_neighbors_10.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.UMAP.3D.n_neighbors_10.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.UMAP.3D.n_neighbors_10.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.UMAP.3D.n_neighbors_10.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.UMAP.3D.n_neighbors_10.html" target="_blank">Open</a></td>
</tr>
<tr>
<th>UMAP.3D.n_neighbors_5</th>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.Significant.Signature.UMAP.3D.n_neighbors_5.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.COV.All.Signature.UMAP.3D.n_neighbors_5.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.ssGSEA.MSigDB.UMAP.3D.n_neighbors_5.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Combat.UMAP.3D.n_neighbors_5.html" target="_blank">Open</a></td>
<td><a href="https://stjudecab_epi.gitlab.io/covidexpressdata/Cluster/GSEA.FPKM.Original.UMAP.3D.n_neighbors_5.html" target="_blank">Open</a></td>
</tr>
</tbody>
</table>
<br>

<table style="border-spacing: 10px;" border="1" class="dataframe table-bordered">
<thead>
<tr style="text-align: center;">
<th>Column/Row</th>
<th>Meaning</th>
</tr>
</thead>

<tbody>
<tr>
<th>Score:ssGSEA</th>
<td>Clustering using single-sample Gene Set Enrichment Analysis Score (<a target="_blank" href="https://www.genepattern.org/modules/docs/ssGSEAProjection">ssGSEA</a>)</td>
</tr>
<tr>
<th>Score:FPKM</th>
<td>Clustering using expression value Fragments Per Kilobase Million(<a target="_blank" href="https://www.rna-seqblog.com/rpkm-fpkm-and-tpm-clearly-explained/">FPKM</a>)</td>
</tr>

<tr>
<th>Data:COV.Significant</th>
<td>ssGSEA score were using genesets from pre-computed differential analysis(<a target="_blank" href="https://doi.org/10.1186/gb-2014-15-2-r29">Using Voom</a>), only genes significant(fold change &gt; 2, False Discovery Rate corrected p-value &lt; 0.05) were considered for each differential analysis.</td>
</tr>


<tr>
<th>Data:COV.All.Signature</th>
<td>ssGSEA score were using genesets from pre-computed differential analysis(<a target="_blank" href="https://doi.org/10.1186/gb-2014-15-2-r29">Using Voom</a>), genes passed different levels of cutoff or top regulated genes were all considered for each differential analysis.</td>
</tr>

<tr>
<th>Data:MSigDB</th>
<td>ssGSEA score were using genesets from Molecular Signatures Database(<a target="_blank" href="http://www.gsea-msigdb.org/gsea/index.jsp">MSigDB</a>)</td>
</tr>

<tr>
<th>Data:BatchCorrected</th>
<td>FPKM values were corrected for batch effect using Combat(<a target="_blank" href="https://bioconductor.org/packages/release/bioc/html/sva.html">From SVA R package</a>, <a target="_blank" href="https://doi.org/10.1093/biostatistics/kxj037">Johnson et al,2007,Biostatistics</a>)</td>
</tr>

<tr>
<th>Data:Original</th>
<td>FPKM values from RSEM method(<a target="_blank" href="https://doi.org/10.1186/1471-2105-12-323">Li and Dewey,2011,BMC Bioinformatics</a>) using STAR(<a target="_blank" href="https://doi.org/10.1093/bioinformatics/bts635">Dobin et al,2013,Bioinformatics</a>)</td>
</tr>

<tr>
<td>–</td>
</tr>

<tr>
<th>PCA.3D</th>
<td>Three dimensions plot(PC1 vs PC2 vs PC3) from <a target="_blank" href="https://en.wikipedia.org/wiki/Principal_component_analysis">Principal component analysis</a>
</td>
</tr>
<tr>
<th>PCA.2PCs</th>
<td>Two dimensions plot(PC1 vs PC2) from <a target="_blank" href="https://en.wikipedia.org/wiki/Principal_component_analysis">Principal component analysis</a>
</td>
</tr>
<tr>
<th>PCA.grid</th>
<td>Two dimensions plots(Many vs Many) from <a target="_blank" href="https://en.wikipedia.org/wiki/Principal_component_analysis">Principal component analysis</a>
</td>
</tr>
<tr>
<th>PCA.components_matrix</th>
<td>Each sample’s projection to top principal components</td>
</tr>
<tr>
<th>PCA.loadings</th>
<td>PCA <a target="_blank" href="https://scentellegher.github.io/machine-learning/2020/01/27/pca-loadings-sklearn.html">loading</a> as each geneset or gene’s loading to top principal components</td>
</tr>
<tr>
<th>PCA.loadings_matrix</th>
<td>PCA <a target="_blank" href="https://scentellegher.github.io/machine-learning/2020/01/27/pca-loadings-sklearn.html">loading matrix</a> as each geneset or gene’s correlations to top principal components</td>
</tr>
<tr>
<th>tSNE.3D</th>
<td>Three dimensions plot from <a target="_blank" href="https://en.wikipedia.org/wiki/T-distributed_stochastic_neighbor_embedding">tSNE analysis</a>
</td>
</tr>
<tr>
<th>tSNE.2D</th>
<td>Two dimensions plot from <a target="_blank" href="https://en.wikipedia.org/wiki/T-distributed_stochastic_neighbor_embedding">tSNE analysis</a>
</td>
</tr>
<tr>
<th>UMAP.3D.n_neighbors_20</th>
<td>Three dimensions plot from <a target="_blank" href="https://umap-learn.readthedocs.io/en/latest">UMAP analysis</a> using parameter n_neighbors = 20</td>
</tr>
<tr>
<th>UMAP.3D.n_neighbors_10</th>
<td>Three dimensions plot from <a target="_blank" href="https://umap-learn.readthedocs.io/en/latest">UMAP analysis</a> using parameter n_neighbors = 10</td>
</tr>
<tr>
<th>UMAP.3D.n_neighbors_5</th>
<td>Three dimensions plot from <a target="_blank" href="https://umap-learn.readthedocs.io/en/latest">UMAP analysis</a> using parameter n_neighbors = 5</td>
</tr>


</tbody>
</table>
<br>
